dependencies.gradle

```diff
@@ -0,0 +1,15 @@
+dependencies {
+    compile("com.github.GTNewHorizons:GT5-Unofficial:unify-build-script-SNAPSHOT:dev") {
+        setChanging(true)
+    }
+    compile("cpw.mods:ironchest:1.7.10-6.0.62.742:deobf")
+    compile("com.github.GTNewHorizons:TinkersConstruct:master-SNAPSHOT:deobf") {
+        setChanging(true)
+    }
+
+    compileOnly("player-api:PlayerAPI:1.7.10:1.4")
+    compileOnly("render-player-api:RenderPlayerAPI:1.7.10:1.4")
+    compileOnly("smart-render:SmartRender:1.7.10:2.1")
+    compileOnly("appeng:appliedenergistics2:rv3-beta-22:dev")
+    compileOnly(name: "CoFHLib", version: "[1.7.10]1.0.3-175-dev", ext: "jar")
```

Comment:
if this is taken from ivy - you should add group id (it won't break anything, but you'll won't get POM parsing error when depending on this). You can call it `ivy` and depend on it like `compileOnly("ivy:CoFHLib:[1.7.10]1.0.3-175-dev:jar")`

Reply:
Awesome! Good to know. Thanks :-)

Reply:
Wait, isn't there an actual maven repo with this lib? Like chicken maven or other (see TiC buildscript.

Maybe we don't even need to get it from ivy

Reply:
CoFHLib is not even needed. In [this](https://github.com/GTNewHorizons/Galacticraft/pull/29#issuecomment-973105370) comment I posted all needed dependencies and their repositories. *I strongly recommend to use these lists.* (http://downloads.gtnewhorizons.com/Mods_for_Jenkins/ is still needed as one dependecy needs AE2)

Reply:
It still won't fix the build issue on my side. Can you build the current state of this repository? Maybe my local files are corrupt

Reply:
Yes, I can. Try temporarily switching to official FG (and back to Gradle 4.4.1)

Reply:
Addendum: These dependecies are indirectly required. In my tests, they were downloaded automatically:
![grafik](https://user-images.githubusercontent.com/35727266/142625480-5dd6ad0b-2696-486e-9791-3c6cc7d8e1ec.png)